keel-core/src/test/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuatorTests.kt

```diff
@@ -42,12 +42,10 @@ import com.netflix.spinnaker.kork.exceptions.SystemException
 import com.netflix.spinnaker.kork.exceptions.UserException
 import dev.minutest.junit.JUnit5Minutests
 import dev.minutest.rootContext
-import io.mockk.coEvery
-import io.mockk.coVerify
-import io.mockk.every
+import io.mockk.coEvery as every
+import io.mockk.coVerify as verify
```

Comment:
Useful trick if you want to not care whether the mock method is a `suspend` method

Reply:
Will the compiler resolve the original `every` and `verify` OK in the case where you have a mix of suspending and non-suspending functions?

Reply:
Yes. Effectively you're _always_ using the suspending ones. It makes no difference when the mock function is non-suspending. I raised this with the library author a while back but he said he wasn't very knowledgeable about coroutines and had assumed there might be some performance penalty. I don't believe that is the case, and even if there were it's unlikely to amount to anything given the number of calls in a unit test scenario.

Reply:
Got it. Yeah, seems like a nice simplification for the author to apply. Thanks for the tip!